### ISSUE TITLE
Migrate to CryEngine V and integrate directly with Renderer

### ISSUE THREAD

**Issue by hendrikp** [hendrikp/CryHTML5#1]
One of the first tasks would be to remove the dependency to the Plugin_D3D which is no longer needed as CryENGINE V is now shipping full engine Sourcecode. So one of the first things would be to extend the CryENGINE V Renderer interface to reliably update/exchange the HTML rendering surface, once this is done the Plugin is ready for its release.

The plugin was already fully working on older CryENGINE versions see here: https://www.youtube.com/watch?v=YphBFbS93EU

This issue and project itself can be taken on by anyone with intermediate C++, DirectX and CryENGINE experience. Happy Coding ;)

**Comment by MergHQ** [comment]
Should we also remove the dependency to Plugin_SDK and build a new system to load the module?

**Comment by hendrikp** [comment]
its not necessary.

2016-03-25 22:24 GMT+01:00 Hugo Holmqvist notifications@github.com:

**Comment by ifight** [comment]
Is there a youtube video showing how to intergrate this with cryengine V?

**Comment by hendrikp** [comment]
no as explained it requires intermediate knowledge.

2016-03-29 16:15 GMT+02:00 ifight notifications@github.com:
